### PR TITLE
Add attributes to radio buttons

### PR DESCRIPTION
**Pull request by monkeypox8**
## Description
Applies attributes to radio buttons. Similar to checkboxes, the most frequent use for this will be with `data` attributes.

Also fixes a small typo and adds some documentation for checkbox properties.

### CODE REVIEW

File: docs/app/views/examples/elements/radio/_preview.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
     disabled: true
   } %>
 
-  <!-- Errror -->
```
Comment (pixelflips) [inline docs/app/views/examples/elements/radio/_preview.html.erb:37]: Nice catch! 😉

**Review by pixelflips — APPROVED** [review]
👍🏼  LGTM! +1

File: docs/lib/sage_rails/app/views/sage_components/_sage_radio.html.erb [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
       sage-radio
       sage-radio--standalone
       <%= component.has_error ? "sage-radio--error" : "" %>
       <%= component.css_classes if component.css_classes.present? %>
     "
+    name="<%= component.name %>"
+    value="<%= component.value %>"
+    <% component.attributes.each do |key, value| %>
+      <%= "#{key}=\"#{value}\"".html_safe %>
+    <% end if component.attributes&.is_a?(Hash) %>
```
Comment (joedakroub) [inline docs/lib/sage_rails/app/views/sage_components/_sage_radio.html.erb:15]: Very nice!

**Review by joedakroub — APPROVED** [review]
Really nice addition. Thanks Kevin! 👍🏼

**Pull request merged**